src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/JsonConverter.Api.sync.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Collections.Generic;
+
+namespace System.Text.Json.Serialization
+{
+    public static partial class JsonConverter
```

Comment:
Then this can be `JsonConverter.Sync.cs` and once we add PipeReader/Writer support, those can be `JsonConverter.Async.PipeReader.cs`.